hazelcast/proxy/queue.py

```diff
@@ -176,7 +187,8 @@ def drain_to(self, target_list: typing.List[ItemType], max_size: int = -1) -> Fu
 
         def handler(message):
             response = queue_drain_to_max_size_codec.decode_response(message)
-            target_list.extend(map(self._to_object, response))
+            items = [self._to_object(item) for item in response]
+            target_list.extend(items)
```

Comment:
I suppose there may be extra list allocation here. map() returns an iterable and extend will use it to extend the existing list without creating a new one. You create another one here. Why do you do this change though?

Reply:
I need to verify that for all the items in the response, there is a schema in local. So, I have to call to_object on each of them before starting to extend the list, to avoid duplicate items in case of SchemaNotFound errors